### ISSUE TITLE
Question No AGOL/Portal

### ISSUE THREAD

**Issue by acbassil** [Esri/geoform-template-js#466]
I just need help seeing how I can implement this functionality.  I do not want to use Portal or AGOL at all if possible.  I do not want to share my feature services with AGOL, (sensitive data) but I want to use the Geoform.  I only want to work with locally hosted feature services from my ArcGis Server.  

I want to have one feature service as the main layer for the geoform.  Plus I want a combo box that queries multiple feature services that populates multiple comboboxes in the geoform.  (So multiple queries)

If an entry already exists, then I want it to prepopulate for quick editing and click submit.

Basically I want to have a page that:

Queries one feature services for a pre-existing name (one combobox, autcomplete)
Queries another seperate feature service for values

User hits submits-and the main feature service is updated based on the data from the 2 others.

So its combining data from multiple feature services into a single feature service, with the added ability to force client side validation through combo/box query dropdown (that is dynamic)

I have very limited knowledge of JS.

**Comment by driskull** [comment]
Hi @acbassil that's going to be tough without some JS knowledge. It's possible, but will require knowledge of querying services, async queries and deferreds. 

You can use the geoform without portal if you would like by using an offline webmap instead.  There is an option for that with the [application boilerplate](https://github.com/Esri/application-boilerplate-js) which this template uses.
